packages/js/src/executors/node/node.impl.ts

```diff
@@ -87,7 +89,11 @@ export async function* nodeExecutor(
     }
   }
 
-  const fileToRun = join(context.root, buildOptions.outputPath, outputFileName);
+  const fileToRun = joinPathFragments(
+    context.root,
+    buildOptions.outputPath,
+    outputFileName.replace(projectRoot, '')
```

Comment:
My only nitpick here is that you could add this to line 86 `mainFolder` since this replace should only happen when `buildOptions.main` path is absolute

Reply:
updated